.clang-format

```diff
@@ -0,0 +1,8 @@
+BasedOnStyle: LLVM
+AlignAfterOpenBracket: AlwaysBreak
+AlignEscapedNewlines: DontAlign
+AllowAllParametersOfDeclarationOnNextLine: false
+PenaltyReturnTypeOnItsOwnLine: 300
+BinPackParameters: false
+ColumnLimit: 100
```

Comment:
I propose to add the following:
UseTab: Never
TabWidth: 2
IndentWidth: 2
NamespaceIndentation: None
IncludeBlocks: Preserve
BreakBeforeTernaryOperators: true
BreakBeforeBinaryOperators: All
PointerAlignment: Middle

Reply:
See https://github.com/Azure/azure-sdk-for-c/pull/42